src/components/ConfigureSupersetEmbeddedDashboardModal/CreateSupersetEmbeddedDashboard.js

```diff
@@ -0,0 +1,118 @@
+import { useDataMutation } from '@dhis2/app-runtime'
```

Comment:
I'm thinking this component should be moved to a subcomponent of DashboardsBar. By convention that is how DashboardsBar components are organized. 

Also, I'm still hoping we can trim down this folder name and component names, and put "Modal" at the end of the Modal components (Create and Update)